### PR TITLE
Crossgen output will not start a new process and pop out a new window

### PR DESCRIPTION
**Pull request by shhsu**
Response to https://github.com/aspnet/dnvm/issues/176

### CODE REVIEW

**Comment by msotclas** [comment]
Hi __@shhsu__, I'm your friendly neighborhood Microsoft Open Technologies, Inc. Pull Request Bot (You can call me MSOTBOT). Thanks for your contribution!
    <p>
        It looks like you're a Microsoft contributor (Peter Hsu). If you're full-time, we DON'T require a Contribution License Agreement. If you are a vendor, please DO sign the electronic Contribution License Agreement. It will take 2 minutes and there's no faxing! https://cla2.msopentech.com.
    </p>

TTYL, MSOTBOT;

**Comment by shhsu** [comment]
@davidfowl

**Comment by glennc** [comment]
@shhsu, can you rebase? Is there something we can do for both clr and coreclr? For now I am just hiding both the windows. But I agree that logging the output in a less annoying manner would be better.

**Comment by shhsu** [comment]
I have done some research. I can't find a way to redirect output for Ngen like I did crossgen. Because Ngen requires admin

* I can only use Start-Process -runas ..., I can't use Invoke-Expression (&)
* -runas option conflicts with -NoNewWindow option.

@anurse, can you chime in?

File: src/dnvm.ps1 [hunk 1]
```diff
@@ -1044,13 +1044,18 @@ function dnvm-install {
         }
         elseif ($Runtime -eq "coreclr") {
             if ($NoNative) {
-              _WriteOut "Skipping native image compilation."
+                _WriteOut "Skipping native image compilation."
```
Comment (glennc) [inline src/dnvm.ps1]: Remove the extra space

**Comment by shhsu** [comment]
Updates?

**Comment by glennc** [comment]
Not sure if there is much you can do for ngen, but no need to block on that we can still merge this one in.

**Pull request merged**